docs/sdk/index.mdx

```diff
@@ -87,3 +91,111 @@ Supported SDKs for server-side:
   </div>
 
 </div>
+
+
+## Features available
```

Comment:
The client and server are different architectures. So, let's create a table for each of them.

You can find the Node JS supported features on this preview page.
https://deploy-preview-42--bucketeer-docs.netlify.app/sdk/server-side/node-js

By the way, the `Destroy` feature is the same feature called `Flushing events` in the client SDK.
In the GO SDK, it has `Close` for the flushing events. Both server-side SDKs have the same features.

Reply:
Done